charms/argo-server/tox.ini

```diff
@@ -33,8 +33,6 @@ allowlist_externals =
     xargs
 commands = 
 ; uses 'bash -c' because piping didn't work in regular tox commands
-  pip-compile requirements.in
-  pip-compile requirements-fmt.in
```

Comment:
Why do we want to remove this ? 

requirements-lint.in is using output from requirements-fmt.txt so first we want to pip compile -fmt and then lint same applies for integration and requirements.in